index.html.unprocessed

```diff
@@ -195,13 +195,14 @@ <h3 class="panel-title">Service Chaining</h3>
 			<!--  Tab Hystrix Dashaboard -->
 			<div role="tabpanel" class="tab-pane active" id="hystrix">
 				<iframe style="padding-top: 5vh; border: 0; width: 100%; height: 80vh; position: inherit;" id="hystrix-dashboard-iframe"
-					src="http://hystrix-dashboard-helloworld-msa.rhel-cdk.10.1.2.2.xip.io/monitor/monitor.html?stream=http%3A%2F%2Fturbine-server%2Fturbine.stream"> </iframe>
+					src="http://HYSTRIXDASHBOARDURLPORT/monitor/monitor.html?stream=http%3A%2F%2Fturbine-server%2Fturbine.stream">
+		</iframe>
```

Comment:
These are the variables in the `index.html.unprocessed` which are replaced in the `CMD` directive of frontend's Dockerfile.